_distro_map.yml

```diff
@@ -6,9 +6,11 @@ openshift-origin:
   site_name: Documentation
   site_url: https://docs.okd.io/
   branches:
-    master-3:
-      name: Latest
+    master:
+      name: Preview
       dir: latest
+      distro-overrides:
+        name: OKD4
```

Comment:
In preview phase we want to highlight the difference between OKD 3 and OKD4 thus distro name is overriden